### DISCUSSION TITLE
upgrade to Typescript

### DISCUSSION THREAD

**Discussion by darckon** [waylaidwanderer/node-chatgpt-api#147] [Discussions]
It is very difficult to pass it to typescript?

**Comment by waylaidwanderer** [comment]
I don't use Typescript. Can you explain what issues you are having?

**Reply by darckon** [reply]
I try to use typescript in the project and it works perfectly, the only problem is that when I want to import a class, for example, a connection to the DB or any class, it doesn't recognize it, I can only import the node_modules libraries

![image](https://user-images.githubusercontent.com/26935900/222724961-5f950888-6419-466c-87a4-b97eab3c75cb.png)

![image](https://user-images.githubusercontent.com/26935900/222725037-01ec4154-897d-460e-b247-2b0fdb851669.png)

![image](https://user-images.githubusercontent.com/26935900/222725575-6ebd519c-58a5-4661-a6c4-a916edb72573.png)

the error : can't find the database.ts module 

And it's not that the route is bad, because if I don't import the chatgpt library it works fine.
